pkg/dbutil/common.go

```diff
@@ -72,8 +72,6 @@ type DBConfig struct {
 	Schema string `toml:"schema" json:"schema"`
 
 	Snapshot string `toml:"snapshot" json:"snapshot"`
-
-	SQLMode string `toml:"sql-mode" json:"sql-mode"`
```

Comment:
I guess other places don't require this `SQLMode` otherwise it won't compile. and in https://github.com/pingcap/tidb-tools/pull/280 this field is used for parser only